src/main/java/org/jenkinsci/plugins/liquibase/evaluator/RollbackBuilder.java

```diff
@@ -41,7 +42,7 @@
     private String rollbackToTag;
     private String rollbackToDate;
 
-    private SimpleDateFormat simpleDateFormat = new SimpleDateFormat(DATE_PATTERN);
+    private transient SimpleDateFormat simpleDateFormat = new SimpleDateFormat(DATE_PATTERN);
```

Comment:
Better to remove the field and to create new instance when you actually need it. Otherwise you may have null here after loading of the builder from the disk